docs/add_metrics.md

```diff
@@ -0,0 +1,156 @@
+[◀︎ Airlift](../README.md) • [◀︎ Getting Started](getting_started.md) • [◀︎ Next Steps](next_steps.md)
+
+## Add Metrics
+
+Airlift incorporates the [jmxutils](https://github.com/martint/jmxutils) library. Exposing
+JMX metrics is very simple.
+
+### Step 1 - Add Needed Dependencies
+
+We need a few additional dependencies. Add the following to the dependencies section of your
+`pom.xml` file:
+
+```xml 
+<dependency>
+    <groupId>org.weakref</groupId>
+    <artifactId>jmxutils</artifactId>
+</dependency>
+
+<dependency>
+    <groupId>io.airlift</groupId>
+    <artifactId>jmx</artifactId>
+</dependency>
+
+<dependency>
+    <groupId>io.airlift</groupId>
+    <artifactId>jmx-http-rpc</artifactId>
+</dependency>
+
+<dependency>
+    <groupId>io.airlift</groupId>
+    <artifactId>jmx-http</artifactId>
+</dependency>
```

Comment:
These are only needed if you want to expose JMX metrics via HTTP, so I'd call that out or simply remove it or move it to a different tutorial.